core/store/src/test/java/uk/gov/gchq/gaffer/store/schema/SchemaTest.java

```diff
@@ -58,19 +57,16 @@
 import static org.junit.Assert.fail;
 import static org.mockito.Mockito.mock;
 
+
 public class SchemaTest {
```

Comment:
Are there any tests for serialising/deserialising JSON schemas containing parent types? Or any examples of this for the wiki?